docs/documentation/ui-rest-api.md

```diff
@@ -172,6 +172,27 @@ Sample response:
 }
 ```
 
+### /api/v1/history/summary (GET)
+
+Returns a list of topology ID submitted by the current user.
```

Comment:
Yes, it will return the true history of topology. I will update the doc to clarify that. Also, a history cleaner will added to cleanup the history for topologies that are older than a threshold ago (one week).